tools/clang/test/CodeGenDXIL/hlsl/shaders/node/mesh-node-no-cross-group-sharing.hlsl

```diff
@@ -0,0 +1,23 @@
+// RUN: %dxc -Tlib_6_9 %s -verify 
+
+// Make sure that cross group sharing is disallowed in mesh node shaders
+
+
+struct MY_MATERIAL_RECORD
+{
+    uint textureIndex;
+    float3 normal;
+};
+
+[Shader("node")]
+[OutputTopology("line")]
+// expected-note@+1{{Launch type defined here}}
+[NodeLaunch("mesh")]
+[NumThreads(4,5,6)]
+[NodeDispatchGrid(2,2,2)]
+void myFancyNode(
+    // expected-error@+1{{'RWDispatchNodeInputRecord' may not be used with mesh nodes (only DispatchNodeInputRecord)}}
+    globallycoherent RWDispatchNodeInputRecord<MY_MATERIAL_RECORD> myProgressCounter 
```

Comment:
Where does it say this is disallowed?  It is listed in the table of input records here:
https://github.com/microsoft/hlsl-specs/blob/main/proposals/0019-mesh-nodes.md#input-record

Reply:
According to https://github.com/microsoft/DirectXShaderCompiler/issues/6479, "writable input records" are not allowed. I assumed RWDispatchNodeInputRecord qualified as a writable input record. Are writable input records allowed? If not, what exactly qualifies as a writable input record?

Reply:
Resolving, since cross group sharing has no indication of not being allowed in the mesh nodes spec.